### PR TITLE
[.NET] Proof-of-concept of removing ObjCRuntime.nfloat in favor of System.Runtime.InteropServices.NFloat.

### PR DESCRIPTION
**Pull request by rolfbjarne**
** DRAFT PR ** PROCEED AT OWN RISK **

* Builds our code.
* Builds monotouch-test for macOS and completes successfully.

This is just to get some CI on this.

### CODE REVIEW

File: dotnet/BreakingChanges.md [hunk 1]
```diff
@@ -26,17 +26,12 @@ favor of the C# 9 `nint` and `nuint` types (these map to `System.IntPtr` and
...
+`System.Runtime.InteropServices.NFloat` type.
 
-* Code that references the `nfloat` type might not compile unless the `ObjCRuntime` namespace is imported.
-
-  Fix: add `using ObjCRuntime` to the file in question.
-
-* Code that references the full typename, `System.nfloat` won't compile.
-
-  Fix: use `ObjCRuntime.nfloat` instead.
+// TODO: explain more
```
Comment (chamons) [inline dotnet/BreakingChanges.md:34]: Reminder before this goes live ^

File: runtime/bindings.m [hunk 2]
```diff
@@ -47,6 +47,33 @@
 	}
 }
 
+xm_nfloat_t
+xamarin_NFloat_objc_msgSend_exception (id self, SEL sel, GCHandle *exception_gchandle)
+{
+	@try {
+#if defined(__i386__)
```
Comment (chamons) [inline runtime/bindings.m:54]: Do we expect NET6 + 32-bit to be a thing?

**Review by chamons — COMMENTED** [review]
Got about 1/2 through when you realized you asked me to review the wrong PR. :)

**Comment by rolfbjarne** [comment]
The final PR is up: #14197.

**Pull request closed without merging**